faust_large_message_serializer/blob_storage/amazon_blob_storage.py

```diff
@@ -0,0 +1,29 @@
+from faust_large_message_serializer.blob_storage.blob_storage import BlobStorageClient
+
+
+class S3UploadException(Exception):
+    pass
+
+
+class AmazonS3BlobStorage(BlobStorageClient):
+    def __init__(self, s3_client):
+        self._s3_client = s3_client
+
+    def delete_all_objects(self, bucket: str, prefix: str) -> None:
+        objects_response = self._s3_client.list_objects_v2(Bucket=bucket, Prefix=prefix)
+        key_objects = [
+            {"Key": object_s3["Key"]} for object_s3 in objects_response["Contents"]
+        ]
+        self._s3_client.delete_objects(Bucket=bucket, Delete={"Objects": key_objects})
```

Comment:
You might need to delete versions as well https://github.com/bakdata/kafka-large-message-serde/blob/master/large-message-core/src/main/java/com/bakdata/kafka/AmazonS3Client.java#L67

Reply:
I am already deleting versions with this https://github.com/bakdata/faust-s3-backed-serializer/pull/7/files#diff-46e96e698abe8e0e238964c69d96389ff11ba50022f75e48602857b3ea722cadR20 https://github.com/bakdata/faust-s3-backed-serializer/pull/7/files#diff-46e96e698abe8e0e238964c69d96389ff11ba50022f75e48602857b3ea722cadR20

Reply:
Ah ok so you can do both with both, nice